packages/lwc-language-server/src/__tests__/component-indexer.test.ts

```diff
@@ -12,9 +12,8 @@ const componentIndexer: ComponentIndexer = new ComponentIndexer({
     workspaceRoot,
 });
 
-beforeEach(async done => {
+beforeEach(async () => {
     await componentIndexer.init();
-    done();
```

Comment:
This started failing with new version of Jest. `beforeEach` expects a function that either returns void (but calls `done`) or returns a `Promise`.